GOVERNANCE.md

```diff
@@ -1,114 +1,159 @@
-	* Emeritus Approvers can be considered "retired" maintainers--while they don't have approval rights, they are listed in the OWNERS file to indicate their expertise in the area.
-* The current list of reviewers and approvers can be found in the [CRI-O OWNERS file](https://github.com/cri-o/cri-o/blob/main/OWNERS).
+The CRI-O project and its leadership embrace the following values:
 
-## Community Member
+* Openness: Communication and decision-making happens in the open and is discoverable for future reference. As much as possible, all discussions and work should take place in public forums and open repositories.
+* Fairness: All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits and not from source.
+* Community over Product or Company: Sustaining and growing our community takes priority over shipping code or sponsors' organizational goals.  Each
+  contributor participates in the project as an individual.
+* Prefer Stable Evolution: A willingness to constantly improve and reevaluate old decisions will keep the project and community relevant and growing.
+  However, this constant growth cannot be done at the expense of stability of the project. New features should be introduced liberally, but must be configured and opted-into until there is proven stability.
+* Inclusivity: We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+* Participation: Responsibilities within the project are earned through
+  participation, and there is a clear path up the contributor ladder into leadership positions.
+ 
+## Contribution Tiers
 
-* Anyone can and is encouraged to be a community member! The CRI-O community eagerly accept PRs, issues, new users and questions.
+CRI-O has four tiers of code contributors: community members, reviewers, approvers and emeritus approvers.
+* Anyone can (and is encouraged to) be a community member, whose responsibilities are up to the contributor (submitting PRs, reviewing PRs).
+* Reviewers are a set of people responsible for reviewing PRs and triaging issues. In most cases, they're also responsible for fixing issues in the code base.
+* Approvers are the set of people that have final say over the direction of the project. For any PR to merge, an approver must approve it.
+* Emeritus Approvers can be considered "retired" approver--while they don't have approval rights, they are listed in the OWNERS file to indicate their expertise in the area.
+The current list of reviewers and approvers can be found in the [CRI-O OWNERS file](https://github.com/cri-o/cri-o/blob/main/OWNERS).
 
-## Reviewer
+### Community Member
 
-### How to become a reviewer
+* Anyone can and is encouraged to be a community member! The CRI-O community eagerly accepts PRs, issues, new users and questions.
```

Comment:
This might be a good place to add also things that you don't mention in reviewer/maintainer sections and encourage users to participate more in the community, and become contributor, like helping with improving documentation, or improving website, evangelize the project by writing blog, etc. CNCF has good examples in [their template under community participant](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md#community-participant)